docs/source/transforms.rst

```diff
@@ -44,6 +44,13 @@ BERTTokenizer
 
    .. automethod:: forward
 
+CharBPETokenizer
+-------------
```

Comment:
Looks like CI is still failing because the underline isn't equal to the title size https://app.circleci.com/pipelines/github/pytorch/text/6981/workflows/1b6e4090-be21-49a9-bca7-84714a050c11/jobs/241539?invite=true#step-105-146